### PR TITLE
Fix DOM nesting on the Storage page dialogs, enable DOM nesting errors

### PR DESCRIPTION
**Pull request by martinpitt**
We have the following cases  in main:

[storage](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-195002-28ebf951-fedora-38-devel-storage/log.html#18):
 - [ ] TestStorageLuks.testLuks: race condition, hard to investigate; not in dialog, wait for storage rewrite, ignore
 - [ ] TestStorageLvm2.test{Lvm,Snapshots}: Grow button is in "dd > div > dd" without a new list wrapper; affected by page reorg, postpone and ignore for now
 - [x] TestStorageMounting.testMounting, TestStorageNfs.testNfsBusy: in "used by pid/service" dialog, let's fix
 - [x] TestStorageLuksDestructive.testLuks1Slots: in dialog, let's fix
 - [x] TestStorageNBDE.test{Basic,RootReboot}: In dialog, let's fix
 - [x] [TestStorage.testUsed](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-storage/log.html#59) - in dialog, let's fix

expensive/networking already got fixed in #19520:
 - [x] expensive: [kdump error dialog](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231019-151349-2a0e10f1-fedora-38-devel-expensive/log.html)
 - [x] networking: [firewall table](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231019-151351-2a0e10f1-fedora-38-devel-networking/log.html#17)

/other is okay.

Builds on top of:
 - [x] PR #19511 (conflicty otherwise)
 - [x] PR #19520 (broken out, intrusive)

### CODE REVIEW

File: pkg/lib/cockpit-components-inline-notification.jsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ InlineNotification.propTypes = {
 export const ModalError = ({ dialogError, dialogErrorDetail, id, isExpandable }) => {
     return (
         <Alert id={id} variant='danger' isInline title={dialogError} isExpandable={!!isExpandable}>
-            { dialogErrorDetail && <p> {dialogErrorDetail} </p> }
+            { typeof dialogErrorDetail === 'string' ? <p>{dialogErrorDetail}</p> : dialogErrorDetail }
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101818-f8bf272a-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/shell/hosts_dialog.jsx [hunk 2]
```diff
@@ -821,7 +821,7 @@ class ChangeAuth extends React.Component {
                                 type="password" value={this.state.login_setup_new_key_password} validated={this.state.login_setup_new_key_password_error ? "error" : "default"} />
                         <FormHelper helperTextInvalid={this.state.login_setup_new_key_password_error} />
                     </FormGroup>
-                    <FormGroup label={_("Confirm new key password")} validated={this.state.login_setup_new_key_password2_error ? "error" : "default"} helperTextInvalid={this.state.login_setup_new_key_password2_error}>
+                    <FormGroup label={_("Confirm new key password")} validated={this.state.login_setup_new_key_password2_error ? "error" : "default"}>
```
Comment (cockpituous) [inline pkg/shell/hosts_dialog.jsx:824]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101818-f8bf272a-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101817-f8bf272a-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/dialogs-common.jsx [hunk 3]
```diff
@@ -248,9 +248,9 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
             dlg = <IpSettingsDialog topic="ipv6" {...properties} />;
 
         if (dlg)
-            resolveDeps(type).then(() => {
-                Dialogs.show(dlg);
-            }).catch(console.error); // not-covered: OS error
+            resolveDeps(type)
+                    .then(() => Dialogs.show(dlg))
```
Comment (cockpituous) [inline pkg/networkmanager/dialogs-common.jsx:252]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101817-f8bf272a-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/wireguard.jsx [hunk 4]
```diff
@@ -117,7 +117,7 @@ export function WireGuardDialog({ settings, connection, dev }) {
                 const key = await cockpit.spawn(["wg", "pubkey"], { err: 'message' }).input(privateKey.trim());
                 setPublicKey(key.trim());
             } catch (e) {
-                console.error(e.message);
+                console.error("Failed to call wg pubkey:", e.message);
```
Comment (cockpituous) [inline pkg/networkmanager/wireguard.jsx:120]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101817-f8bf272a-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/shell/hosts_dialog.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/shell/hosts_dialog.jsx:824]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101817-f8bf272a-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113059-f8bf272a-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/shell/hosts_dialog.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/shell/hosts_dialog.jsx:824]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113059-f8bf272a-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/dialogs-common.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/networkmanager/dialogs-common.jsx:252]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/wireguard.jsx (hunk 4 above)
Comment (cockpituous) [inline pkg/networkmanager/wireguard.jsx:120]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/shell/hosts_dialog.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/shell/hosts_dialog.jsx:824]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/sosreport/sosreport.jsx [hunk 5]
```diff
@@ -177,7 +177,9 @@ function sosCreate(args, setProgress, setError, setErrorDetail) {
     });
 
     task.catch(error => {
-        console.error(JSON.stringify(error)); // easier investigation of failures, errors in pty mode may be hard to see
+        // easier investigation of failures, errors in pty mode may be hard to see
+        if (error.problem !== 'cancelled')
+            console.error("Failed to call sos report:", JSON.stringify(error));
```
Comment (cockpituous) [inline pkg/sosreport/sosreport.jsx:182]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
After round 1: The ListingTable nesting fix [caused some test regressions](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101817-f8bf272a-fedora-38-devel-other/log.html#120-2), selectors need to be adjusted. Some of the [storage failures are also due to that](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-101818-f8bf272a-fedora-38-devel-storage/log.html#32-2), and some are independent problems. /expensive and /networking are green now.

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-195003-28ebf951-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-194951-28ebf951-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-194951-28ebf951-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-195002-28ebf951-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-194951-28ebf951-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-195005-28ebf951-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231023-194951-28ebf951-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102436-6e7b9faf-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 6]
```diff
@@ -622,15 +622,13 @@ export const TangKeyVerification = ({ url, adv }) => {
...
-                    {cmd}
-                </ClipboardCopy>
-            </Text>
+            <span>{_("In a terminal, run: ")}</span>
+            <ClipboardCopy hoverTip={_("Copy to clipboard")}
+                            clickTip={_("Successfully copied to clipboard!")}
+                            variant="inline-compact"
+                            isCode>
+                {cmd}
+            </ClipboardCopy>
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:631]: These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102436-6e7b9faf-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 7]
```diff
@@ -692,21 +690,19 @@ const RemovePassphraseField = (tag, key, dev) => {
...
+                                isChecked={val !== false}
+                                label={_("Confirm removal with an alternate passphrase")}
+                                onChange={(_event, checked) => change(checked ? "" : false)}
+                                body={val === false
+                                    ? <p className="slot-warning">
+                                        {_("Removing a passphrase without confirmation of another passphrase may prevent unlocking or key management, if other passphrases are forgotten or lost.")}
+                                    </p>
+                                    : <FormGroup label={_("Passphrase from any other key slot")} fieldId="remove-passphrase">
+                                        <TextInputPF id="remove-passphrase" type="password" value={val} onChange={(_event, value) => change(value)} />
+                                    </FormGroup>
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:703]: These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102436-6e7b9faf-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx [hunk 8]
```diff
@@ -1257,7 +1257,8 @@ export const StopProcessesMessage = ({ mount_point, users }) => {
     return (
         <div className="modal-footer-teardown">
             { process_rows.length > 0
-                ? <p>{fmt_to_fragments(_("The mount point $0 is in use by these processes:"), <b>{mount_point}</b>)}
+                ? <>
+                    <p>{fmt_to_fragments(_("The mount point $0 is in use by these processes:"), <b>{mount_point}</b>)}</p>
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1261]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102436-6e7b9faf-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx [hunk 9]
```diff
@@ -1268,15 +1269,16 @@ export const StopProcessesMessage = ({ mount_point, users }) => {
...
                 : null
             }
             { process_rows.length > 0 && service_rows.length > 0
                 ? <br />
                 : null
             }
             { service_rows.length > 0
-                ? <p>{fmt_to_fragments(_("The mount point $0 is in use by these services:"), <b>{mount_point}</b>)}
+                ? <>
+                    <p>{fmt_to_fragments(_("The mount point $0 is in use by these services:"), <b>{mount_point}</b>)}</p>
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1281]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102436-6e7b9faf-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 6 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:631]: These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 7 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:703]: These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 8 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1261]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 9 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1281]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042825-94ae7a2d-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042836-94ae7a2d-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 10]
```diff
@@ -692,21 +690,19 @@ const RemovePassphraseField = (tag, key, dev) => {
...
-                                      : <FormGroup label={_("Passphrase from any other key slot")} fieldId="remove-passphrase">
-                                          <TextInputPF id="remove-passphrase" type="password" value={val} onChange={(_event, value) => change(value)} />
-                                      </FormGroup>
-                                  }
-                        />
-                    </Form>
+                    <Checkbox id="force-remove-passphrase"
+                                isChecked={val !== false}
+                                label={_("Confirm removal with an alternate passphrase")}
+                                onChange={(_event, checked) => change(checked ? "" : false)}
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:696]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042836-94ae7a2d-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 6 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:631]: These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 7 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:703]: These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx [hunk 11]
```diff
@@ -1107,27 +1107,27 @@ const UsersPopover = ({ users }) => {
             bodyContent={
                 <>
                     { services.length > 0
-                        ? <p>
-                            <b>{_("Services using the location")}</b>
+                        ? <>
+                            <p><b>{_("Services using the location")}</b></p>
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1111]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx [hunk 12]
```diff
@@ -1107,27 +1107,27 @@ const UsersPopover = ({ users }) => {
...
                     }
                     { services.length > 0 && processes.length > 0
                         ? <br />
                         : null
                     }
                     { processes.length > 0
-                        ? <p>
-                            <b>{_("Processes using the location")}</b>
+                        ? <>
+                            <p><b>{_("Processes using the location")}</b></p>
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1125]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 8 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1261]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 9 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1281]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-inline-notification.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-inline-notification.jsx:93]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 6 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:631]: These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 7 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:703]: These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 11 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1111]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 12 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1125]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 8 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1261]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 9 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1281]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042828-94ae7a2d-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 10 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:696]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-060544-c71e1521-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
Rawhide failure is unrelated, see PR #19525

File: pkg/storaged/crypto-keyslots.jsx [hunk 13]
```diff
@@ -622,15 +622,13 @@ export const TangKeyVerification = ({ url, adv }) => {
             <Text component={TextVariants.p}>{_("Check the key hash with the Tang server.")}</Text>
 
             <Text component={TextVariants.h3}>{_("How to check")}</Text>
-            <Text component={TextVariants.p}>
-                {_("In a terminal, run: ")}
-                <ClipboardCopy hoverTip={_("Copy to clipboard")}
-                               clickTip={_("Successfully copied to clipboard!")}
-                               variant="inline-compact"
```
Comment (mvollmer) [inline pkg/storaged/crypto-keyslots.jsx:629]: Hmm, shouldn't `inline-compact` turn the ClipboardCopy into a span?

Comment (mvollmer) [reply pkg/storaged/crypto-keyslots.jsx:629]: Ah, no, it changes it to `display: inline`, but keeps it a div.

Comment (martinpitt) [reply pkg/storaged/crypto-keyslots.jsx:629]: Yes, but the more important point is that the `<p>` was a block element, and by default the next element gets rendered below it. `<span>` renders the next element to the right.

**Review by mvollmer — APPROVED** [review]

**Pull request merged**